### ISSUE TITLE
Extend all Posix-Wrapper unit tests with direct posix calls

### ISSUE THREAD

**Issue by elfenpiff** [eclipse-iceoryx/iceoryx#824]
## Brief feature description

The current posix wrapper are tested in a way that only the wrapper itself is used and tested with itself. If possible, we should call the posix calls directly in one part of a tests and verify that the posixWrapper  behaves as intended when it works on the structure.

**Comment by elBoberido** [comment]
Yes, this is kind of bootstrapping our implementation. One call should be done directly to the POSIX function, e.g. `send` to test `receive`. After that, the wrapper can be used to test itself.
